101-app-configuration/prereqs/prereq.azuredeploy.json

```diff
@@ -0,0 +1,37 @@
+{
+    "$schema": "https://schema.management.azure.com/schemas/2015-01-01/deploymentTemplate.json#",
+    "contentVersion": "1.0.0.0",
+    "parameters": {
+      "configStoreName": {
+        "type": "string",
+        "metadata": {
+          "description": "Specifies the name of the app configuration store."
+        }
+      },
+      "location": {
+        "type": "string",
+        "defaultValue": "westus",
+        "metadata": {
+          "description": "Specifies the Azure location where the app configuration store should be created."
+        }
+      },
+      "skuName": {
+        "type": "string",
+        "defaultValue": "free",
+        "metadata": {
+          "description": "Specifies the SKU of the app configuration store."
+        }
+      }
+    },
+    "resources": [
```

Comment:
key-value pairs have to be added manually after the store is created - they can't be added through the template.

Reply:
ok, it looks like we'll have to create one in the test sub and use that, otherwise deployment will always fail.

Can you add detail to the readme about which keys/values need to be added for this particular sample?  I'll create a store and add those values.

You can remove the prereq templates, but would be good to have a sample template for creating a config store (as a separate sample) if you want to add that...

Reply:
Hi @bmoore-msft - I think we're ready for the store to be added. I've updated the README with the info and confirmed locally that it works - although location matters, so the SKU and Size in the App Config store should be available in whatever the resource group location is.